src/main/java/org/apache/commons/collections4/IterableUtils.java

```diff
@@ -554,6 +555,34 @@
         };
     }
 
+    /**
+     * Provides iteration over the elements contained in a pair of Iterables in-tandem.
+     * <p>
+     * The returned iterable has an iterator that traverses the elements in {@code a}
+     * and {@code b} together until one of the iterables is traversed completely.
+     * <p>
+     * The returned iterable's iterator does NOT support {@code remove()}.
+     *
+     * @param <L> the left elements' type
+     * @param <R> the right elements' type
+     * @param left the iterable for the left side elements
+     * @param right the iterable for the right side elements
+     * @return an iterable, over the decorated iterables to traverse them together until one is
+     * exhausted
+     * @throws NullPointerException if any iterator is null
+     */
```

Comment:
New public and protected method should have `@since 4.5` Javadoc tags.

Reply:
Done